client/src/app.js

```diff
@@ -1,20 +1,46 @@
 import React from 'react';
 import styled from 'styled-components';
-import './app.css';
-
-const Main = styled.div`
-  margin: 40px auto;
-  max-width: 650px;
-  line-height: 1.6;
-  font-size: 18px;
-  color: #444;
-  padding: 0 10px;
-`;
-
-const App = () => (
-  <Main>
-    <h1>Hello, world?</h1>
-  </Main>
-);
+import Review from './components/review.js';
+
+class App extends React.Component {
+  constructor(props) {
```

Comment:
Doesn't look like you ever use props. You can get leave the constructor and the super arguments blank if you don't intend on using props

Reply:
Thanks, I cut out the reference to props.  Appreciate it.